docs/redis.md

```diff
@@ -1,3 +1,71 @@
+# `redis` Metrics Receiver
+
+The redis receiver can retrieve stats from your redis server through the [INFO](https://redis.io/commands/info) command. 
+
+
+## Configuration
+
+Following the guide for [Configuring the Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/configuration#file-location), add the required elements for your redis configuration.
+
+To configure a receiver for your redis metrics, specify the following fields:
+
+| Field                 | Default                   | Description |
+| ---                   | ---                       | ---         |
+| `type`                | required                  | Must be `redis`. |
+| `endpoint`            | `localhost:6379`          | The url exposed by redis |
+| `collection_interval` | `60s`                     | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`. |
+| `password`            |                           | The password used to connect to the server. |
+
+Example Configuration:
+
+```yaml
+metrics:
+  receivers:
+    redis_metrics:
+      type: redis
+      endpoint: localhost:6379
+      collection_interval: 30s
+      password: pwd
+  service:
+    pipelines:
+      redis_pipeline:
+        receivers:
+          - redis_metrics
+```
+
+## Metrics
+
+The Ops Agent collects the following metrics from your redis servers.
+
+| Metric                                                              | Data Type | Unit    | Labels  | Description    |
+| ---                                                                 | ---    | ---     | ---     | ---            | 
+| workload.googleapis.com/redis/uptime	                              | sum	   | s		   |         | Number of seconds since Redis server start |
```

Comment:
Our normal convention is that uptime metrics should be gauges, since users care about the value and not the rate. I'm not sure how that intersects with OT standards.

Reply:
I'm not sure what the meaning is here. it seems as though, because it's a sum it is just the value of the amount of seconds.  So, it's just the total amount of time it's been up not the rate of time that it's increasing by.